client/pcr.go

```diff
@@ -33,7 +33,7 @@ func min(a, b int) int {
 	return b
 }
 
-// Get a list of selections corresponding to the TPM's implemented PCRs
```

Comment:
very nit: This function would be nice to export, so I'd like to settle on a proper name

Looks like implemented PCR means the TPM supports certain set of PCR indexes with a hash algorithm, whereas allocated means the TPM has actually memory for the use of those PCRs. It's subtle, but we change the behavior here to actually return the allocated PCRs not implemented. So, let's change the name accordingly: `allocatedPCRs`